bylaws/cf-council.md

```diff
@@ -2,67 +2,123 @@
+  - [Regular and Special Meetings](#regular-and-special-meetings)
+  - [Electronic Participation and Action by Written Consent](#electronic-participation-and-action-by-written-consent)
+  - [Agendas, Access, and Records](#agendas-access-and-records)
+- [Composition and Membership](#composition-and-membership)
+  - [Tenure, Transition and Apportioning Duties](#tenure-transition-and-apportioning-duties)
+  - [CFC Chairperson](#cfc-chairperson)
+  - [CFC Offices](#cfc-offices)
+- [Meeting Procedures and Decision-Making](#meeting-procedures-and-decision-making)
+  - [Quorum for Meetings](#quorum-for-meetings)
+  - [Decision-Making Process](#decision-making-process)
+  - [Matters Requiring Supermajority Vote](#matters-requiring-supermajority-vote)
+- [Committees and Delegation](#committees-and-delegation)
 
-The CFC convenes regularly and holds an Annual General Meeting to review the annual report, financial status, and set future plans.
-Special general meetings may be initiated with a defined agenda if proposed by at least a third of the CFC members or by a significant fraction of the general members.
+## Meetings, Participation, and Record Keeping
 
-Agendas and schedules for these gatherings will be released in advance.
-Meetings open to the broader public are accessible to all members and interested observers.
-Summaries of key decisions and discussions will be shared with the community to ensure that all members are informed and can provide feedback.
+### Regular and Special Meetings
 
-- **Concerns and Actions**: Members can voice concerns or propose actions by emailing `council@mailing-list` (TODO: or initiating a GitHub discussion, etc.).
+- **Meeting Frequency:** The CFC holds regular meetings at least once per quarter to discuss strategic and operational matters.
+- **Annual General Meeting (AGM):** An AGM reviews the annual report, financial status, and sets plans for the upcoming year.
+- **Special General Meetings:** Convened for specific purposes, proposed by at least one-third of CFC members or a significant fraction of the general membership.
+- **Place and Time:** Meetings can be held at any time and place as determined by the CFC, with a preference for online videoconference or teleconference.
 
-## Composition and Membership
-
-The CFC is composed of elected members who are recognized for their individual contributions and expertise in the open-source community. While members may be affiliated with various organizations, they are expected to prioritize the interests of the CF community in their decisions and actions.
+### Electronic Participation and Action by Written Consent
 
-CFC members are required to disclose any potential conflicts of interest, especially those related to their employment. This disclosure ensures transparency and helps maintain the integrity of CFC decisions.
+- **Remote Involvement:** CFC members may participate in meetings electronically, including like teleconferencing or video conferencing.
```

Comment:
Not quite sure what you're trying to say with "including like teleconferencing or video conferencing"

Reply:
Yes. I'll update wording. It's "interesting" when you start looking at bylaws, especially for older organizations, how unclear it was whether or not you can join remotely.

But I did update this in the notice page as well (to prefer remote comms).